lib/covalence/helpers/shell_interpolation.rb

```diff
@@ -6,7 +6,7 @@ class ShellInterpolation
 
       def self.parse_shell(input)
         Covalence::LOGGER.info "Evaluating requested interpolation: \"#{input}\""
-        matches = input.scan(/.?\$\([^)]*\)/)
+        matches = input.scan(/.?\$\([^)]*\)+/)
```

Comment:
The imperfect fix for the time being.

Reply:
Means no spaces allowed right? No big deal but should probably put it as a caveat or known limitation in the README
[Example with spaces](http://rubular.com/r/OLOlS7ZREY)

Reply:
This would help address the #64, but not resolve it due to easy ways of escaping the regex, as @disaac highlighted.